### ISSUE TITLE
Can't load plugin

### ISSUE THREAD

**Issue by DanHunting** [johnpholden/qgis3redistricter#3]
I'm a long-time ArcGIS user, but somewhat new to QGIS, so it's somewhat likely that I'm doing something wrong here.
I've downloaded _qgis3redistricter-1.0-beta.zip_.
Select Install from ZIP from the Manage and Install Plugins menu.
The following error message is generated when I install:
![image](https://user-images.githubusercontent.com/70179281/91107008-0d55dd00-e629-11ea-815c-7fced9ee10a6.png)
However, Statto Redistricter QGISv3 is listed as installed after the error window is closed.
When I click on the plugin, the same error message appears.
I'm running QGIS 3.14 now, results were the same in v3.12

**Comment by johnpholden** [comment]
Hi,

I haven't seen this one before - it looks to me as though the plugin didn't
install successfully, in spite of the positive result. There may be
something going on with newer QGIS versions where it doesn't like the
period in 1.0, since that's where the name of the plugin truncates.

If you could, would you mind uninstalling, renaming the zip to
qgisv3redistricter-master.zip, reinstalling, and letting me know if the
same error message appears? I can't upgrade my QGIS for another couple
weeks, so this would be very helpful.

Sorry for the issue and please feel free to reach out if you have any other
QGIS questions

Thanks
John

**Comment by DanHunting** [comment]
Tried the rename - no joy - same error message. I've also switched to QGIS v3.10. Any other suggestions?

**Comment by DanHunting** [comment]
Hi John -
I'm still getting the same error message. I've tried it on three computers, including a brand-new one that I just bought today. (The ultimate clean install - took it out of the box, turned it on, installed QGIS, then the plug in!) I'd like to make sure that I'm doing everything correctly: 
- Download qgis3redistricter-1.0-beta.zip
- Open QGIS, go to the plugins menu
- Select Install from ZIP from the Manage and Install Plugins menu
- Select qgis3redistricter-1.0-beta.zip

Is this right?

**Comment by johnpholden** [comment]
That looks correct to me, yes. I plan to work on this next week with a
clean QGIS install.

Thanks
John

**Comment by johnpholden** [comment]
I took the following steps to try to recreate the problem:
1. made a minor update to the experimental branch following feedback regarding a different error
2. downloaded a fresh QGISv3.14 for Windows
3. downloaded the experimental plugin from https://github.com/johnpholden/qgis3redistricter/tree/experimental and selecting Code>Download as ZIP
4. opened QGIS and tried to install from zip, and it worked.

I just realised you're probably downloading the beta release - the experimental release is the most current and while I can't guarantee it's stable I'm working on making that a full release this week.
